daphne/src/lib.rs

```diff
@@ -126,6 +126,11 @@ pub enum DapAbort {
     #[error("invalidBatchInterval")]
     InvalidBatchInterval,
 
+    /// Invalid DAP version. Sent in response to requests that use a DAP version URL endpoint
+    /// that does not match to the version set in TaskConfig or simply does not exist.
```

Comment:
This mechanism isn't specific to versioned URLs:
```suggestion
    /// Invalid DAP version. Sent in response to requests for an unsupported (or unknown) DAP version.
```